sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -500,7 +500,14 @@ public Node visitAddGeneratedColumnDefinition(SqlBaseParser.AddGeneratedColumnDe
             visit(context.columnConstraint(), ColumnConstraint.class));
     }
 
-    // Assignments
+    @Override
+    public Node visitAlterTableOpenClose(SqlBaseParser.AlterTableOpenCloseContext context) {
+        return new AlterTableOpenClose(
+            (Table) visit(context.alterTableDefinition()),
+            context.OPEN() != null
+        );
+    }
+// Assignments
```

Comment:
It just delineates the 2 assignment visits below it (it was there in master), but since both functions have "assignment" in their function names it is not too useful. Can remove.